spec/feature/registeration_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'capybara/rspec'
+
+feature "Chitter user behaviour" do
+
+  scenario "Users can sign up for Chitter" do
+    visit '/'
+    click_link "Sign Up"
+
+    expect { sign_up }.to change(User, :count).by(1)
+    expect(User.count).to eq(1)
```

Comment:
This is expectation tests the same outcome as the previous line.  The previous line is the better test.